src/components/Footer.tsx

```diff
@@ -0,0 +1,83 @@
+import {
+  Box,
+  Button,
+  chakra,
+  Container,
+  Stack,
+  Text,
+  useColorModeValue,
+  VisuallyHidden
+} from "@chakra-ui/react";
+import { FaGithub, FaInstagram, FaFacebook } from "react-icons/fa";
+import { ReactNode, useState } from "react";
+
+const SocialButton = ({
+  children,
+  label,
+  href
+}: {
+  children: ReactNode;
+  label: string;
+  href: string;
+}) => {
+  return (
+    <Button
+      bg={useColorModeValue("gray.100", "gray.900")} px={4}
+      rounded={"full"}
+      w={10}
+      h={10}
+      cursor={"pointer"}
+      as={"a"}
+      href={href}
+      display={"inline-flex"}
+      alignItems={"center"}
+      justifyContent={"flex-end"}
+      transition={"background 0.3s ease"}
+      _hover={{
+        bg: useColorModeValue("blackAlpha.200", "whiteAlpha.200")
+      }}>
+      <VisuallyHidden>{label}</VisuallyHidden>
+      {children}
+    </Button>
+  );
+};
+
+export default function Footer() {
+  return (
+    <Box
+      bg={useColorModeValue("gray.50", "gray.900")}>
+      <Container
+        as={Stack}
+        maxW={"6xl"}
+        py={4}
+        direction={{ base: "column", md: "row" }}
+        spacing={4}
+        justify={{ base: "center", md: "space-between" }}
+        align={{ base: "center", md: "center" }}>
+        <Text>
+          © 2023 Olivier. All rights reserved
+        </Text>
+         
+        <Stack direction={"row"} justify={"right"} spacing={8}>
+          <Box as="a" href={"/OurTeam"}>
+            MEET THE TEAM
+          </Box>
+          <Box as="a" href={"/Contact"}>
+            CONTACT
+          </Box>
+        </Stack>
+        <Stack direction={"row"} spacing={6}>
+          <SocialButton label={"Github"} href={"#"}>
+            <FaGithub />
```

Comment:
NON-BLOCKING: Is there a reason you didn't use Chakra's third party library icon approach? See [here](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams/get) for more info.

Reply:
Also, at least for this icon, you could go ahead and link to the front end GitHub repo.